managedtenants/data/selectorsyncset.yaml.j2

```
@@ -62,7 +62,8 @@ items:
         {{ maybe_annotations(ADDON.metadata.get('commonAnnotations')) | indent(8) }}
         {{ maybe_labels(ADDON.metadata.get('commonLabels')) | indent(8) }}
       spec:
-        displayName: {{ADDON.metadata['name']}}
+        displayName: |-
```

```suggestion
        displayName: >|
```
Needs to be folded or we might have newlines in that string

---

Let's go ahead with `>-` then to chomp any possible newline chars in the end.

---

Ah sorry, copy-paste mistake. `>-` is right